prepare-lua

```diff
@@ -2,19 +2,29 @@
 
 . ./config
 
+if [ -z "${VARIETY}" ]; then
+  VARIETY="5.1"
+fi
+
 git clone https://github.com/kaitai-io/kaitai_struct_lua_runtime "$LUA_RUNTIME_DIR"
 
 # Preinstall hererocks, lua installer
 pip install hererocks
 
 # Install lua
-hererocks lua_install -r^ -l5.3
+hererocks lua_install -r^ -l${VARIETY}
 export PATH=$PATH:$PWD/lua_install/bin
 
 # Check available versions
 lua -v
-luarocks -v
+luarocks --version
 
 # Install Lua prerequisites
 luarocks install luaunit
 luarocks install luafilesystem
+
+# Install luabitop only for Lua 5.1 target
+# 5.2 and 5.3 targets have built-in present
```

Comment:
What does it mean to _have built-in present_? Is it supposed to mean 🕥 (yeah, my clock shows 22:30 currently 😆), 🎁 or it's another word with typo?

Please spend half a minute more next time reading the sentences you write, so that I don't have to waste my time figuring out what you mean.

Reply:
It's a sentence "Install luabitop only for Lua 5.1 target, 5.2 and 5.3 targets have built-in present", context is fairly obvious to me, but whatever, if you're gonna nitpick on such things, it'll become harder for you to get anything done